OracleDatabase/dockerfiles/12.1.0.2/Dockerfile.ee

```diff
@@ -75,6 +79,8 @@ RUN sed -i -e "s|###ORACLE_EDITION###|EE|g" $INSTALL_DIR/$INSTALL_RSP &&
 USER oracle
 
 RUN cd $INSTALL_DIR       && \
```

Comment:
If we want to squash the big intermediate layers, which should be removed indeed, shouldn't we either:
- convert once and only once e.g. via the build.sh the 2 install zip files to one big .tar.gz file, that we can add and untar at the right location via a simple ADD command here?
- or be patient for docker 1.13, which will add the ability (albeit in experimental mode) to squash all layers into one? :) (in the meantime, you can docker run <yourBigImage> true then docker export | docker import it, and add metadata again, but I agree it's tedious)

That way, we wouldn't have to impose a new restriction (sudo) on that build system...